cocos/2d/CCLabel.h

```diff
@@ -83,6 +83,9 @@ typedef struct _ttfConfig
     }
 }TTFConfig;
 
+class Sprite;
+class SpriteBatchNode;
+
 /**
  * @brief Label is a subclass of SpriteBatchNode that knows how to render text labels.
```

Comment:
Please modify the class comments, The Label is not a subclass of SpriteBatchNode any more. 
So the comments must be updated, otherwise it will be misleading.